pkg/operator/v1helpers/core_getters.go

```diff
@@ -100,3 +116,26 @@ func (g combinedSecretInterface) List(opts metav1.ListOptions) (*corev1.SecretLi
 	}
 	return ret, nil
 }
+
+func validateGetOptions(options metav1.GetOptions) error {
```

Comment:
do we really need this method? we could just call `validateResourceVersion` directly.